ci-operator/config/openshift/osin/openshift-osin-master.yaml

```diff
@@ -0,0 +1,33 @@
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: "7"
+binary_build_commands: go build ./...
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+promotion:
```

Comment:
what image is this promoting? Aren't you missing config for image build here?

Reply:
I blatantly copied this from `apiserver-library-go` as it's also a library. If this section is not meant to be here, shall I remove it from there as well (in a different PR, perhaps)?

Reply:
I suppose the automation brought it back. It doesn't hurt to leave it there.

Reply:
We use "promotion to ocp:4.X" to recognize official repos for the tooling that do branching, fast-forwards, private org mirrors etc, so it is desirable to have this if this should be involved in all these processes.